src/pybind/mgr/dashboard/frontend/cypress/integration/a11y/navigation.e2e-spec.ts

```diff
@@ -0,0 +1,20 @@
+import { NavigationPageHelper } from '../ui/navigation.po';
+describe('Navigation accessibility', () => {
```

Comment:
then you don't have to supply it in each test cases.
```suggestion
describe('Navigation accessibility', { retries: 0 }, () => {
```

Reply:
@nizamial09  I don't quite understand what to do here. Could you please explain some more

Reply:
yup, you put the `{retries: 0}` individually in each test cases right. like this 
`it('top-nav should have no accessibility violations', { retries: 0 }, () => {`

Instead of that, if you put it just in the `describe` part, you don't need to put the `retries` in each of the test cases. The one in the `describe` will be applied to the entire block. 
https://docs.cypress.io/guides/guides/test-retries#Test-Suite-s

Reply:
Thanks a bunch!